.circleci/config.yml

```diff
@@ -23,6 +23,12 @@ jobs:
             for script in $(ls scripts/*.sh); do
                 shellcheck -x $script
             done
+      - run:
+          name: lint-pre-scripts
+          command: |
+            for script in $(find incubator stable -type f -name "*.sh"); do
```

Comment:
there may be other scripts in these folders for various reasons that are part of the helm chart.  I don't think we want to lint those, so maybe a more specific match would be good here?

Reply:
Changed to only check pre-test-script.sh in any chart's ci folder.